Tests/BrowserServicesKitTests/ContentBlocker/ContentBlockerRulesManagerInitialCompilationTests.swift

```diff
@@ -240,20 +295,21 @@ final class ContentBlockerRulesManagerInitialCompilationTests: XCTestCase {
         let expRecompiled = CountedFulfillmentTestExpectation(description: "New Rules Compiled")
         rulesUpdateListener.onRulesUpdated = { _ in
             expRecompiled.fulfill()
-        }
 
-        if expRecompiled.currentFulfillmentCount == 1 { // finished compilation after cold start (using last compiled rules)
-
-            mockLastCompiledRulesStore.onRulesGet = {}
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.etag, oldEtag)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.name, mockRulesSource.ruleListName)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.trackerData, mockRulesSource.trackerData?.tds)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.identifier, oldIdentifier)
-        } else if expRecompiled.currentFulfillmentCount == 2 { // finished recompilation of rules due to changed tds
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.etag, updatedEtag)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.name, mockRulesSource.ruleListName)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.trackerData, mockRulesSource.trackerData?.tds)
-            XCTAssertEqual(mockLastCompiledRulesStore.rules.first?.identifier, newIdentifier)
+            if expRecompiled.currentFulfillmentCount == 1 { // finished compilation after cold start (using last compiled rules)
```

Comment:
This was not needed for my work, but I noticed that code wasn’t getting called. I moved it up into the block. Please check that this is correct.